.gitignore

```diff
@@ -13,7 +13,7 @@ pids
 
 # Directory for instrumented libs generated by jscoverage/JSCover
 lib-cov
-data/
+/data/
```

Comment:
This may be wrong - but it prevented the `reference/data` directory from being in git.